hedera-node/src/main/java/com/hedera/services/txns/TransitionRunner.java

```diff
@@ -57,7 +58,8 @@
 			TokenAccountWipe,
 			TokenCreate,
 			CryptoTransfer,
-			TokenDelete
+			TokenDelete,
+			TokenUpdate
```

Comment:
nit: Place `TokenDelete` and `TokeUpdate` after the `TokenCreate` for consistency